libs/sdk-ui-tests-e2e/package.json

```diff
@@ -47,13 +47,19 @@
         "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long cypress/",
         "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
         "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
-        "cypress": "node ./scripts/run_cypress_local.js"
+        "cypress": "node ./scripts/run_cypress_local.js",
+        "run-isolated": "docker-compose -f docker-compose-isolated.yaml up --abort-on-container-exit --exit-code-from isolated-tests --force-recreate --always-recreate-deps --renew-anon-volumes",
+        "docker-build-local": "docker build --file Dockerfile_local -t gooddata-ui-sdk .",
+        "run-isolated-local": "yarn docker-build-local && IMAGE_ID=gooddata-ui-sdk yarn run-isolated",
```

Comment:
Please use `npm run docker-build-local` instead of `yarn docker-build-local` in package.json scripts. This project should not depend on yarn being installed.